Test flake?

```
I0730 17:52:38.838585   22906 decoder.go:141] decoding stream as JSON
I0730 17:52:49.993936   22906 trace.go:57] Trace "Create *api.Policy" (started 2015-07-30 17:52:39.351684674 +0000 UTC):
[3.940809ms] [3.940809ms] About to create object
[10.642215891s] [10.638275082s] END
--- FAIL: TestTemplate-16 (14.00s)
panic: interface conversion: interface is nil, not *api.Role [recovered]
	panic: interface conversion: interface is nil, not *api.Role
goroutine 108 [running]:
testing.func·006()
	/usr/local/go/src/testing/testing.go:441 +0x181
github.com/openshift/origin/pkg/authorization/registry/role.(*storage).CreateRole(0xc2093822a0, 0x2ad147b346f0, 0xc209dca270, 0xc209a54f70, 0x16, 0x0, 0x0)
	/home/travis/gopath/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/authorization/registry/role/registry.go:56 +0x101
github.com/openshift/origin/pkg/cmd/server/admin.func·017(0xc208fc8180, 0x0, 0x0)
	/home/travis/gopath/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/cmd/server/admin/overwrite_bootstrappolicy.go:164 +0x1149
github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource.func·003(0xc208fc8180, 0x0, 0x0)
```

--- a/pkg/authorization/registry/clusterrole/registry.go
+++ b/pkg/authorization/registry/clusterrole/registry.go
@@ -53,11 +53,18 @@ func (s *storage) ListClusterRoles(ctx kapi.Context, label labels.Selector, fiel
 
 func (s *storage) CreateClusterRole(ctx kapi.Context, node *authorizationapi.ClusterRole) (*authorizationapi.ClusterRole, error) {
 	obj, err := s.Create(ctx, node)
+	if err != nil {
+		return nil, err
+	}
+
 	return obj.(*authorizationapi.ClusterRole), err
 }
 
 func (s *storage) UpdateClusterRole(ctx kapi.Context, node *authorizationapi.ClusterRole) (*authorizationapi.ClusterRole, bool, error) {
 	obj, created, err := s.Update(ctx, node)
+	if err != nil {
+		return nil, created, err
+	}
 	return obj.(*authorizationapi.ClusterRole), created, err
 }
 

--- a/pkg/authorization/registry/clusterrolebinding/registry.go
+++ b/pkg/authorization/registry/clusterrolebinding/registry.go
@@ -58,11 +58,17 @@ func (s *storage) ListRoleBindings(ctx kapi.Context, label labels.Selector, fiel
 
 func (s *storage) CreateRoleBinding(ctx kapi.Context, node *authorizationapi.RoleBinding) (*authorizationapi.RoleBinding, error) {
 	obj, err := s.Create(ctx, node)
+	if err != nil {
+		return nil, err
+	}
 	return obj.(*authorizationapi.RoleBinding), err
 }
 
 func (s *storage) UpdateRoleBinding(ctx kapi.Context, node *authorizationapi.RoleBinding) (*authorizationapi.RoleBinding, bool, error) {
 	obj, created, err := s.Update(ctx, node)
+	if err != nil {
+		return nil, created, err
+	}
 	return obj.(*authorizationapi.RoleBinding), created, err
 }
 

